Sources/swift-build/--show-dependencies.swift

```diff
@@ -0,0 +1,75 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import PackageType
+import struct PackageDescription.Version
+
+final class ShowDependencies {
```

Comment:
Do we need `ShowDependencies ` class here?  I think we can make `dumpDependenciesOf` a free function. It would be nicer to call it. `ShowDependencies.dumpDependenciesOf(` vs `dumpDependenciesOf(`